### DISCUSSION TITLE
some questions about read xlsx file last column

### DISCUSSION THREAD

**Discussion by Ggggggguy** [qax-os/excelize#1649] [Q&A]
Found that using github.com/xuri/exceliz to read xlsx files, there are several lines of the last column without data directly reported the wrong panic, have you encountered?

**Comment by xuri** [comment]
Hi @Ggggggguy, thanks for your feedback. I have tested with lasted version and it works well. Which version of the Go language and Excelize library are you using? Could you show us a complete, standalone example program or reproducible demo?
